notifications/rest/notifications/send-notification-detailed/send-notification-detailed.7.x.java

```diff
@@ -27,7 +27,7 @@ public static void main(String[] args) {
                 .setApn("{\"aps\" : " +
                         "{ \"alert\": " +
                         "{\"title\":\"New alert\"," +
-                        "\"body\" : \"Hello Bob!\"}")
+                        "\"body\" : \"Hello Bob!\"}}}")
```

Comment:
@devidliik could we use [JSON.stringify](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) instead of concatenating a string?

Reply:
this is .java

Reply:
lol my bad didn't notice the file extension!